### PR TITLE
Implemented function which returns list of all cities, function which…

### PR DESCRIPTION
**Pull request by kunduzb17**
… returns list of currencies and updated database

### CODE REVIEW

File: app/src/main/java/com/immi/thebrogrammers/immi/MainActivity.kt [hunk 1]
```diff
@@ -11,7 +11,7 @@ import kotlinx.android.synthetic.main.activity_main.*
 class MainActivity : AppCompatActivity() {
 
   lateinit var toolbar: Toolbar
-
+  val url = "https://www.numbeo.com/api/currency_exchange_rates?api_key=1yj7w26vkf03z6"
```
Comment (AnuarTB) [inline app/src/main/java/com/immi/thebrogrammers/immi/MainActivity.kt]: Is this variable used in MainActivity?

File: app/src/main/java/com/immi/thebrogrammers/immi/MainActivity.kt [hunk 2]
```diff
@@ -28,6 +28,7 @@ class MainActivity : AppCompatActivity() {
       android.R.layout.simple_list_item_1,
       geoNames)
     searchBar.setAdapter(adapter)
+
```
Comment (AnuarTB) [inline app/src/main/java/com/immi/thebrogrammers/immi/MainActivity.kt]: Could you please remove empty line?

File: app/src/main/java/com/immi/thebrogrammers/immi/ImmIParser.kt [hunk 3]
```diff
@@ -30,4 +32,23 @@ object ImmIParser {
...
+    val gson = GsonBuilder().create()
+    val homefeed = gson.fromJson(body, CurrFeed::class.java)
+    return homefeed
+  }
+
+
+}
+
+class CitiesFeed(val cities: List<City>)
+class CurrFeed(val exchange_rates: List<Currency>)
```
Comment (AnuarTB) [inline app/src/main/java/com/immi/thebrogrammers/immi/ImmIParser.kt]: Please add newline at the end of file :)

**Comment by kunduzb17** [comment]
793704852b578f7538cc81714baddc71858b295c I have done this commit!

**Pull request merged**